web/client/src/models/file.ts

```diff
@@ -52,6 +52,10 @@ export class ModelFile extends ModelArtifact<InitialFile> {
     return this.type === 'model'
   }
 
+  get fingerprint(): string {
```

Comment:
Isn't a fingerprint typically a hash of content?

Reply:
yes, it is hash without running hash function :) did not want to bring another lib to just use it for one use case